io.jor.bugdom.yml

```diff
@@ -31,8 +31,7 @@ modules:
       - install -D ./io.jor.bugdom.metainfo.xml ${FLATPAK_DEST}/share/metainfo/io.jor.bugdom.metainfo.xml
 
 finish-args:
-  - --socket=wayland
-  - --socket=fallback-x11
+  - --socket=x11
```

Comment:
missing --share=ipc

Reply:
@hfiguiere I didn't write the flatpak manifest, I'm just fixing a bug. The original manifest didn't have the IPC permission either.

Reply:
@hfiguiere I updated it, is it okay now?